### PR TITLE
feat: Add intel and AMD/Xilinx as partners

### PR DESCRIPTION
**Pull request by ZacharySabourin**
Closes https://github.com/openhwgroup/openhwgroup.org/issues/504

Closes https://github.com/openhwgroup/openhwgroup.org/issues/505

Closes https://github.com/openhwgroup/openhwgroup.org/issues/507

### CODE REVIEW

File: data/members.yml [hunk 1]
```diff
@@ -1,490 +1,408 @@
...
-         link: https://www.nortonrosefulbright.com/en-ca
-         img_width: 200
-         level: 2
-       -
-         image: /images/partners/rbc-logo-shield.svg
-         title: RBC
-         link: https://www.rbc.com/about-rbc.html
-         img_width: 60
-         level: 2
+      - image: /images/partners/amd-xilinx.png
```
Comment (autumnfound) [inline data/members.yml:273]: Amazon comes before AMD in alphabetic order. Small nitpick

Comment (ZacharySabourin) [reply data/members.yml]: LOL whoops. Updated

**Review by autumnfound — APPROVED** [review]

**Comment by autumnfound** [comment]
Let me know when all issues have +1 from Rick and I'll merge this in

**Comment by ZacharySabourin** [comment]
@autumnfound He's approved the changes

**Pull request merged**